site_root/input/resources/extensions/StructureDefinition-drs-url.json

```diff
@@ -0,0 +1,76 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "drs-url",
```

Comment:
I am wondering if it needs to be `drs-url` or `drs-uri` and I am leaning toward `drs-uri` due to:

- By definition: [Comparing URI vs URL](https://www.keycdn.com/support/comparing-uri-vs-url) seems to be a good post describing the conceptual difference between URI and URL. While being included in URI, a URL should identify the network location of a resource. So, the prefix of a URL should be a network protocol. `drs://` rather seems to be a name space and we will need an additional client service to interpret what `drs` means (based on the DRS specification)
- By specification: [the DRS documentation](https://ga4gh.github.io/data-repository-service-schemas/preview/release/drs-1.0.0/docs/) favors "URI" over "URL" when it comes to "DRS URI"

Reply:
`uri` is fine  (went with `url` based on the slack conversation)

Reply:
I agree that it's actually URI and prefer `drs-uri` as well